### PR TITLE
Update images, remove old images, remove adoptopenjdk images

### PR DESCRIPTION
**Pull request by fredg02**

### CODE REVIEW

File: gtk3-wm/debian-metacity/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ARG FROM_TAG=""
-FROM eclipsecbi/debian:${FROM_TAG}
+FROM debian:${FROM_TAG}
```
[the remark this answers is not in the document]
Comment (heurtematte) [reply gtk3-wm/debian-metacity/Dockerfile:2]: uid_entrypoint is still useful when using ssh in a container. Prevent from  "No user exists for uid $someUid" 

https://access.redhat.com/solutions/4665281

[the remark this answers is not in the document]
Comment (fredg02) [reply gtk3-wm/debian-metacity/Dockerfile:2]: As Mikael found out, `uid_entrypoint` is not needed for SSH in docker images running on OKD. See https://ci.eclipse.org/cbi/job/test-jobs/job/ssh_in_docker_wo_uidentrypoint/ for an example (that fails, but for a different reason).

**Review by mbarbero — APPROVED** [review]
Globally approve, I'll let you discuss about the hugo version with webdev.

File: gtk3-wm/debian-metacity/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (heurtematte) [reply gtk3-wm/debian-metacity/Dockerfile:2]: Looks like my build works with your image fr3d/openssh-client:0.0.1 but not with eclipsecbi/openssh-client:latest where I had to explicitly call the entry point to make it works.

With:  fr3d/openssh-client:0.0.1
https://gitlab.eclipse.org/eclipsefdn/it/releng/gitlab-runner-service/gitlab-ci-templates/-/jobs/329771

With: eclipsecbi/openssh-client:latest
https://gitlab.eclipse.org/eclipsefdn/it/releng/gitlab-runner-service/gitlab-ci-templates/-/jobs/329772

I also found something weird with docker scout: docker scout compare --to eclipsecbi/openssh-client:latest fr3d/openssh-client:0.0.1

There is a number of packages where versions differ like: 
alpine-baselayout       apk   3.4.3-r2              3.1.2-r0
openssh                 apk   9.6_p1-r0             8.0_p1-r0     

Looks like eclipsecbi/openssh-client:latest is not par of the build process and have never been updated.

**Pull request merged**